app/Http/Controllers/Twill/FeeController.php

```diff
@@ -2,47 +2,77 @@
+        $this->disablePermalink();
+        $this->disablePublish();
+        $this->enableEditInModal();
+        $this->setModuleName('fees');
+        $this->setTitleColumnKey('fee_category.title');
+        $this->setTitleColumnLabel('Category');
     }
 
-    public function index($parentModuleId = null): mixed
+    protected function additionalIndexTableColumns(): TableColumns
     {
-        $feeCategories = FeeCategory::ordered()->get();
-        $feeAges = FeeAge::ordered()->get();
-
-        return view('twill.fees.index', [
-            'customForm' => true,
-            'editableTitle' => false,
-            'customTitle' => 'Update admission fees',
-            'form_fields' => $this->fees->getFormFields(),
-            'feeCategories' => $feeCategories,
-            'feeAges' => $feeAges,
-            'saveUrl' => route('twill.visit.fees.update'),
-            'publish' => false,
-        ]);
+        $columns = new TableColumns();
+        $columns->add(
+            Relation::make()
+                ->field('title')
+                ->title('Age')
+                ->relation('fee_age')
+        );
+
+        $columns->add(
+            Text::make()
+                ->title('Price')
+                ->field('price')
+                ->optional()
+        );
+
+        return $columns;
     }
 
-    public function update($id, $submoduleId = null): \Illuminate\Http\JsonResponse
-    {
-        if (array_key_exists('cancel', request()->all())) {
-            return redirect()->route('twill.visit.fees.update');
-        }
+    // protected FeeRepository $fees;
 
-        $this->fees->update(request()->except('_token'));
+    // public function __construct(FeeRepository $fees)
+    // {
+    //     $this->fees = $fees;
+    //     $this->request = request();
+    // }
 
-        $this->fireEvent();
+    // public function index($parentModuleId = null): mixed
+    // {
+    //     $feeCategories = FeeCategory::ordered()->get();
+    //     $feeAges = FeeAge::ordered()->get();
 
-        return redirect()->route('twill.visit.fees.update');
-    }
+    //     return view('twill.fees.index', [
+    //         'customForm' => true,
+    //         'editableTitle' => false,
+    //         'customTitle' => 'Update admission fees',
+    //         'form_fields' => $this->fees->getFormFields(),
+    //         'feeCategories' => $feeCategories,
+    //         'feeAges' => $feeAges,
+    //         'saveUrl' => route('twill.visit.fees.update'),
+    //         'publish' => false,
+    //     ]);
+    // }
+
+    // public function update($id, $submoduleId = null): \Illuminate\Http\JsonResponse
+    // {
+    //     if (array_key_exists('cancel', request()->all())) {
+    //         return redirect()->route('twill.visit.fees.update');
+    //     }
+
+    //     $this->fees->update(request()->except('_token'));
+
+    //     $this->fireEvent();
+
+    //     return redirect()->route('twill.visit.fees.update');
+    // }
```

Comment:
SonarQube is mad about this commented out code:
https://sonarcloud.io/project/issues?issueStatuses=OPEN%2CCONFIRMED&pullRequest=682&id=art-institute-of-chicago_artic.edu